pkg/progress/progressreader.go

```diff
@@ -37,7 +37,7 @@ func (p *Reader) Read(buf []byte) (n int, err error) {
 		}
 	}
 	if p.current-p.lastUpdate > updateEvery || err != nil {
-		p.updateProgress(err != nil && read == 0)
+		p.updateProgress(err == io.EOF)
```

Comment:
How about just `err != nil`? We want the newline even in cases where the progress bar doesn't reach the end.